[feature] Segment analytics implementation

alex-deaconu opened the pull request:
New CH version
Removed event tracking; moved pageview tracking to CH

alex-deaconu commented:
@olegrise please review. Thanks!

olegrise commented on src/com/risevision/ui/client/common/controller/CommonHeaderController.java:
```diff
@@ -35,7 +35,15 @@ public static int getContentHeight() {
+	
+	private static native void trackPageviewNative(String contentId) /*-{
+		$wnd.angular.element($wnd.document.getElementsByTagName("common-header")).scope().trackPageview(contentId);
+	}-*/;
```
Wow, this is some ninja style coding :sunglasses:

olegrise commented on war/Rdn2ui.html:
```diff
@@ -67,27 +57,6 @@
-<!--	<script type="text/javascript" charset="utf-8">-->
-<!--	  var is_ssl = ("https:" == document.location.protocol);-->
-<!--	  var asset_host = is_ssl ? "https://s3.amazonaws.com/getsatisfaction.com/" : "http://s3.amazonaws.com/getsatisfaction.com/";-->
-<!--	  document.write(unescape("%3Cscript src='" + asset_host + "javascripts/feedback-v2.js' type='text/javascript'%3E%3C/script%3E"));-->
```
@alex-deaconu this is not related to segment analytics. Is it?

olegrise commented:
:+1: jut one comment related to removing getsatisfaction script.

alex-deaconu commented:
@olegrise I've refactored the spaghetti code, good call. With regards to the getSatisfaction script, that was a test we performed many years ago in the RVA, I just removed the commented out code. 

Thanks.
